### ISSUE TITLE
Document the billing webhooks after sign-up

### ISSUE THREAD

**Issue by pixernot** [KeypDemoCorp/cooking-app-frontend#306]
Accessibility should be considered when implementing this.

This will unblock the upcoming reporting feature.
